libs/ballot-interpreter-nh/src/accuvote.test.ts

```diff
@@ -0,0 +1,62 @@
+import { unsafeParse } from '@votingworks/types';
+import { assert } from '@votingworks/utils';
+import { join } from 'path';
+import {
+  decodeFrontTimingMarkBits,
+  findTimingMarks,
+  scanForTimingMarksByScoringBlocks,
+} from './accuvote';
+import { withSvgDebugger } from './debug';
+import { readGrayscaleImage } from './images';
+import { decodeBottomRowTimingMarks } from './timing_marks';
+import { FrontMarksMetadataSchema } from './types';
+
+test('hudson p1 front', async () => {
```

Comment:
This is essentially the integration test for this library. Scans a real (template) image and locates the marks, decodes the barcodes, etc.